src/blockfilter.h

```diff
@@ -59,7 +59,7 @@ class GCSFilter
     explicit GCSFilter(const Params& params = Params());
 
     /** Reconstructs an already-created filter from an encoding. */
-    GCSFilter(const Params& params, std::vector<unsigned char> encoded_filter);
+    GCSFilter(const Params& params, std::vector<unsigned char> encoded_filter, bool filter_checked=false);
```

Comment:
Clang format for each of these defaults
```suggestion
    GCSFilter(const Params& params, std::vector<unsigned char> encoded_filter, bool filter_checked = false);
```

Reply:
No longer needed as the default value was removed